README.md

```diff
@@ -1,10 +1,10 @@
-redux-promise
-=============
+# redux-promise
 
-[![build status](https://img.shields.io/travis/acdlite/redux-promise/master.svg?style=flat-square)](https://travis-ci.org/acdlite/redux-promise)
+[![build status](https://img.shields.io/travis/redux-utilities/redux-promise/master.svg?style=flat-square)](https://travis-ci.org/redux-utilities/redux-promise)
```

Comment:
Can you remove `flat-square` style so we have it aligned like the others?

Reply:
Done